src/resources/views/crud/fields/select2_from_ajax.blade.php

```diff
@@ -212,10 +212,21 @@ function bpFieldInitSelect2FromAjaxElement(element) {
         // when one of those dependencies changes value
         // reset the select2 value
         for (var i=0; i < $dependencies.length; i++) {
-            $dependency = $dependencies[i];
-            $('input[name='+$dependency+'], select[name='+$dependency+'], checkbox[name='+$dependency+'], radio[name='+$dependency+'], textarea[name='+$dependency+']').change(function () {
-                element.val(null).trigger("change");
-            });
+            //if element has name it means is not in repeatable, because in repeatable we strip the names out.
```

Comment:
```suggestion
            $dependency = $dependencies[i];
            //if element has name it means is not in repeatable, because in repeatable we strip the names out.
```

Reply:
This was causing an error because the variable was not defined